src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -49,13 +49,6 @@ Copyright (c) .NET Foundation. All rights reserved.
    <_ShortFrameworkVersion>$(TargetFramework.Substring($(_ShortFrameworkIdentifier.Length)))</_ShortFrameworkVersion>
   </PropertyGroup>
 
-  <!-- Map short name to long name. See earlier comment for example of how to work with identifiers that are not recognized here. -->
```

Comment:
Note the !contains(+) above here. This was meant to filter out PCL and will bite for net5 platform specific tfms.

Reply:
Will it bite net5 platform specific TFMs?  They will be something like `net5.0-ios14`, so there won't be a `+` character.

Reply:
Nevermind. I remember + was discussed at some point and for some reason that's what I remembered. lol